public/app/features/dashboard-scene/utils/interactions.ts

```diff
@@ -1,16 +1,25 @@
 import { config, reportInteraction } from '@grafana/runtime';
+import { DashboardCreatedProps, DashboardInitProps } from 'app/features/dashboard-scene/utils/tracking';
 
 let isScenesContextSet = false;
 
 export const DashboardInteractions = {
   // Dashboard interactions:
-  dashboardInitialized: (properties?: Record<string, unknown>) => {
+  dashboardInitialized: (properties: DashboardInitProps) => {
     reportDashboardInteraction('init_dashboard_completed', { ...properties });
   },
 
+  dashboardCopied: (properties: DashboardCreatedProps) => {
+    reportInteraction('grafana_dashboard_copied', properties);
+  },
+
+  dashboardCreatedOrSaved: (name: 'created' | 'saved', properties: DashboardCreatedProps) => {
+    reportDashboardInteraction(name, properties, 'grafana_');
```

Comment:
I don't think this approach works :-/. The original event name was `grafana_dashboard_` while this one will cause `grafana_dashboards_` (note the plural for `dashboard`).